packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -115,7 +115,7 @@ export const bm25s = (model: ModelData): string[] => [
 retriever = BM25HF.load_from_hub("${model.id}")`,
 ];
 
-export const chatterbox = (model: ModelData): string[] => [
```

Comment:
not related to this PR but fixes lint in PR (introduced in https://github.com/huggingface/huggingface.js/pull/1503)